Flow.Launcher.Core/Plugin/JsonRPCPluginSettings.cs

```diff
@@ -81,6 +83,7 @@ public void UpdateSettings(IReadOnlyDictionary<string, object> settings)
                             break;
                     }
                 }
+                Save();
```

Comment:
maybe put this out of the loop. No need to save at every setting change?

Reply:
Oh whoops! 

I meant to make sure it was after the loop too!